app/src/main/java/com/futurice/freesound/feature/home/HomeFragment.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2016 Futurice GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.futurice.freesound.feature.home;
+
+import com.futurice.freesound.R;
+import com.futurice.freesound.core.BindingBaseFragment;
+import com.futurice.freesound.inject.fragment.BaseFragmentModule;
+import com.futurice.freesound.utils.Preconditions;
+import com.futurice.freesound.viewmodel.Binder;
+import com.squareup.picasso.Picasso;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.Unbinder;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.schedulers.Schedulers;
+import polanski.option.AtomicOption;
+import timber.log.Timber;
+
+import static butterknife.ButterKnife.bind;
+import static com.futurice.freesound.utils.Preconditions.get;
+import static io.reactivex.android.schedulers.AndroidSchedulers.mainThread;
+
+public final class HomeFragment extends BindingBaseFragment<HomeFragmentComponent> {
+
+    @Nullable
+    @Inject
+    HomeFragmentViewModel homeFragmentViewModel;
+
+    @Nullable
+    @Inject
+    Picasso picasso;
+
+    @Nullable
+    @BindView(R.id.avatar_image)
+    ImageView avatarImage;
+
+    @Nullable
+    @BindView(R.id.username)
+    TextView userName;
+
+    @Nullable
+    @BindView(R.id.about)
+    TextView about;
+
+    @NonNull
+    private final AtomicOption<Unbinder> unbinder = new AtomicOption<>();
+
+    @NonNull
+    private final Binder binder = new Binder() {
+
+        @Override
+        public void bind(@NonNull final CompositeDisposable d) {
```

Comment:
For me there is no point to do `disposable.add(homeFragmentViewModel.getImage()`. Way too long